docs/en/glossary/glossary.asciidoc

```diff
@@ -566,8 +566,7 @@ cells or regions to analyze patterns across multiple dimensions. See
 
 [[glossary-hidden-index]] hidden data stream or index::
 <<glossary-data-stream,Data stream>> or <<glossary-index,index>> excluded from
-most <<glossary-index-pattern,index patterns>> by default. See
-{ref}/multi-index.html#hidden[Hidden data streams and indices].
+most <<glossary-index-pattern,index patterns>> by default.
```

Comment:
This content is still around. It just got moved with https://github.com/elastic/elasticsearch/pull/74529. Can we link to the new page instead?
```suggestion
most <<glossary-index-pattern,index patterns>> by default. See
/api-conventions.html#multi-hidden[Hidden data streams and indices].
```